otelcli/plumbing.go

```diff
@@ -128,11 +130,20 @@ func grpcOptions() []otlpgrpc.Option {
 	if config.Insecure || (isLoopbackAddr(config.Endpoint) && !strings.HasPrefix(config.Endpoint, "https")) {
 		grpcOpts = append(grpcOpts, otlpgrpc.WithInsecure())
 	} else if !isInsecureSchema(config.Endpoint) {
-		var tlsConfig *tls.Config
+		tlsConfig := &tls.Config{}
 		if config.NoTlsVerify {
-			tlsConfig = &tls.Config{
-				InsecureSkipVerify: true,
+			diagnostics.InsecureSkipVerify = true
+			tlsConfig.InsecureSkipVerify = true
+		}
+		if config.Certificate != "" {
```

Comment:
I'm not sure this is the correct behavior and need to verify, then add the same or a helper for https.